.github/workflows/build-and-push.yml

```
@@ -52,6 +50,14 @@ jobs:
           username: ${{ github.repository_owner }}
           password: ${{ secrets.GITHUB_PAT }}
 
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: "ghcr.io/${{ inputs.docker_image_repository }}"
+          tags: |
+            type=sha,value=latest,enable=true,priority=100,prefix=sha-,suffix=,format=short
```

```suggestion
            type=sha,enable=true,priority=100,prefix=sha-,suffix=,format=short
            type=raw,value=latest
```

Shouldn't it be like this? 

https://github.com/docker/metadata-action/blob/b2391d37b4157fa4aa2e118d643f417910ff3242/README.md

---

ooohh i misunderstood the doc to mean that only for branches based on their example